srcpkgs/xremap/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'xremap'
+pkgname=xremap
+version=0.4.4
+revision=1
+build_style=cargo
+short_desc="Dynamic key remapper for X11 and Wayland"
+maintainer="akhil <akhil.j@outlook.in>"
+license="MIT"
+homepage="https://github.com/k0kubun/xremap"
+changelog="https://raw.githubusercontent.com/k0kubun/xremap/master/CHANGELOG.md"
+distfiles="https://github.com/k0kubun/xremap/archive/refs/tags/v${version}.tar.gz"
+checksum=323cc9a900e7bc226008179619e49ee679caa6c9ade85e69bb5e710ffb0ed7ff
+
+vlicense() {
+	vlicense LICENSE
+}
```

Comment:
`vlicense()` isn't a function recognized by `xbps-src`. If you would try to build this package yourself, you would see:
```
/home/coder/git/void-packages/srcpkgs/xremap/template: line 14: syntax error near unexpected token `('
/home/coder/git/void-packages/srcpkgs/xremap/template: line 14: `vlicense() {'
=> ERROR: xbps-src: failed to read /home/coder/git/void-packages/srcpkgs/xremap/template!
```
```suggestion
post_install() {
	vlicense LICENSE
}
```